content/uiux/concepts/crazy-eights/crazy-eights.md

```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Crazy Eights'
+Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights's** a Design Sprint method: meaning it is a quick sketching exercise that challenges people to sketch eight different ideas in eight minutes. The goal is to push beyond your first idea, which is usually basic(the least innovative), and to generate a wide variety of solutions to your initial design.
```

Comment:
```suggestion
**Crazy Eights** is a Design Sprint method used as a quick sketching exercise that challenges people to sketch eight distinct ideas in eight minutes, for the problem at hand. The goal is to push beyond the preliminary idea, and to generate a wide variety of solutions.
```
We refrain from using any personal pronouns like you, we, .. in the entry.

Reply:
done